### PR TITLE
Fix/slack notification errors

### PR DESCRIPTION
**Pull request by vigneswararaomacharla**
Fixed traceback notification error messages to slack channel.

### CODE REVIEW

File: hammer/library/aws/ecs.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def check(self, task_definitions=None):
                         logging.error(f"Access denied in {self.account} "
                                       f"(ecs:{err.operation_name})")
                     else:
-                        logging.exception(f"Failed to describe task definitions in {self.account} "
```
Comment (bakalor) [inline hammer/library/aws/ecs.py]: Do we need to add "AccessDeniedException" error code into ClientError exception as was made to elasticsearch.py module?

File: hammer/library/aws/ecs.py [hunk 2]
```diff
@@ -198,7 +198,7 @@ def check(self, task_definitions=None):
...
+                        logging.error(f"Failed to describe task definitions in {self.account} "
```
Comment (MrBakalo) [inline hammer/library/aws/ecs.py:201]: Do we need to add "AccessDeniedException" error code into ClientError exception as was made to elasticsearch.py module?

File: hammer/reporting-remediation/reporting/create_ecs_external_image_source_issue_tickets.py [hunk 3]
```diff
@@ -66,18 +66,6 @@ def create_tickets_ecs_external_images(self):
                             bu=bu, product=product,
                         )
                         IssueOperations.set_status_closed(ddb_table, issue)
-                    # issue.status != IssueStatus.Closed (should be IssueStatus.Open)
```
Comment (MrBakalo) [inline hammer/reporting-remediation/reporting/create_ecs_external_image_source_issue_tickets.py:69]: Why this section was removed from this and five more modules?

File: hammer/library/aws/ecs.py (hunk 2 above)
Comment (vigneswararaomacharla) [reply hammer/library/aws/ecs.py:201]: Here in this case, we have inactive task definitions, which are failing to describe.

File: hammer/reporting-remediation/reporting/create_ecs_external_image_source_issue_tickets.py (hunk 3 above)
Comment (vigneswararaomacharla) [reply hammer/reporting-remediation/reporting/create_ecs_external_image_source_issue_tickets.py:69]: Here in this case, we don't have option to modify/change existing resources (no remediation also). So deleted these code blocks (which Oleksii suggested for as code review for previous issues) which are not required to validate and sent notification for change.

**Review by MrBakalo — APPROVED** [review]

**Pull request merged**